bigtable/README.MD

```diff
@@ -0,0 +1,39 @@
+# Google Cloud Bigtable via Apache HBase
+This initialization action installs Apache HBase libraries and the [Google Cloud Bigtable](https://cloud.google.com/bigtable/) [HBase Client](https://github.com/GoogleCloudPlatform/cloud-bigtable-client).
+
+
+## Using this initialization action
+You can use this initialization action to create a Dataproc cluster configured to connect to Cloud Bigtable:
+
+1. Create a Bigtable instance by following [these directions](https://cloud.google.com/bigtable/docs/creating-instance).
+2. Using the `gcloud` command to create a new cluster with this initialization action.
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+    --initialization-actions gs://dataproc-initialization-actions/bigtable/bigtable.sh \
+    --metadata bigtable-instance=<BIGTABLE INSTANCE>
+    ```
+3. When it is finished being created the cluster will have HBase libraries and the Bigtable Client.
+4. You can validate the deployment by SSHing to the master (`gcloud compute ssh <CLUSTER_NAME>-m`) and running `hbase shell`.
+
+## Running example MR job on cluster
+1. Example code can be get from [cloud-bigtable-examples](https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/tree/master/java/dataproc-wordcount).
+2. Run compilation with: 
```

Comment:
We've started using `1.` for each numbered bullet point -- the markdown compiler will magically turn them into the right number. That way, if you delete a number, you don't need to change the others.

Reply:
Thanks for that hint ;)